src/compute-pr-actions.ts

```diff
@@ -136,28 +136,40 @@ function canBeMergedNow(info: PrInfo): boolean {
         return false;
     }
 
-    return hasFinalApproval(info);
+    return hasFinalApproval(info).approved;
 }
 
-function hasFinalApproval(info: PrInfo): boolean {
+function hasFinalApproval(info: PrInfo) {

```

Comment:
Used to return a `boolean` - now returns a boolean and a message about who has the access rights to pass the check